contrib/experimental/great_expectations_experimental/expectations/expect_column_values_confidence_to_be_greater_than_or_equal_to_threshold_for_data_label.py

```diff
@@ -0,0 +1,136 @@
+"""
+This is a template for creating custom ColumnMapExpectations.
+For detailed instructions on how to use it, please see:
+    https://docs.greatexpectations.io/docs/guides/expectations/creating_custom_expectations/how_to_create_custom_column_map_expectations
+"""
+
+import dataprofiler as dp
+import json
+import numpy as np
+# remove extra tf loggin
+import tensorflow as tf
+tf.compat.v1.logging.set_verbosity(tf.compat.v1.logging.ERROR)
+
+from great_expectations.execution_engine import (
+    PandasExecutionEngine,
+    SparkDFExecutionEngine,
+    SqlAlchemyExecutionEngine,
+)
+from great_expectations.expectations.expectation import ColumnMapExpectation
+from great_expectations.expectations.metrics import (
+    ColumnMapMetricProvider,
+    column_condition_partial,
+)
+
+
+class ColumnValuesConfidenceForDataLabelToBeGreaterThanOrLessThanThreshold(ColumnMapMetricProvider):
+    """MetricProvider Class for Data Label Probability greater than \
+    or equal to the user-specified threshold"""
+    
+    # This is the id string that will be used to reference your metric.
+    condition_metric_name = "column_values.prediction_confidence_for_data_label_greater_than_or_equal_to_threshold"
+
+    condition_value_keys = ("threshold", "data_label",)
+
+    # This method implements the core logic for the PandasExecutionEngine
+    @column_condition_partial(engine=PandasExecutionEngine)
+    def _pandas(cls, column, threshold, data_label, **kwargs):
+        """
+        Implement the yes/no question for the expectation
+        """
+        labeler = dp.DataLabeler(labeler_type='structured')
+        try:
+            results = labeler.predict(column, predict_options={"show_confidences": True})
+        except:
+            results = None
```

Comment:
How should this be handled?

Reply:
I actually lean towards taking this out entirely. I did add a `raise` on the `data_label` parameter so its more explicit what a user can submit. But this error should be handled by DP or Great Expectations when it runs.